openedx/features/course_experience/api/v1/tests/test_views.py

```diff
@@ -25,17 +27,22 @@ def setUp(self):  # pylint: disable=arguments-differ
         # Need to supply tracker name for the EventTestMixin. Also, EventTestMixin needs to come
         # first in class inheritance so the setUp call here appropriately works
         super().setUp('openedx.features.course_experience.api.v1.views.tracker')
+        self.course = CourseFactory.create(self_paced=True, start=timezone.now() - datetime.timedelta(days=1000))
 
     def test_reset_deadlines(self):
-        CourseEnrollment.enroll(self.user, self.course.id, CourseMode.VERIFIED)
+        enrollment = CourseEnrollment.enroll(self.user, self.course.id, CourseMode.VERIFIED)
+        enrollment.schedule.start_date = timezone.now() - datetime.timedelta(days=100)
```

Comment:
I changed days to 1000 here because of [this check](https://github.com/openedx/edx-platform/blob/656ec5def9e211ca6b41f5ec3fd5764dd7dc3c3e/openedx/core/djangoapps/schedules/utils.py#L65).